src/commonTest/kotlin/fr/acinq/lightning/channel/states/SpliceTestsCommon.kt

```diff
@@ -155,32 +176,41 @@ class SpliceTestsCommon : LightningTestSuite() {
     @Test
     @OptIn(ExperimentalCoroutinesApi::class)
     fun `splice to purchase inbound liquidity -- not enough funds`() {
-        val (_, bob) = reachNormal(aliceFundingAmount = 100_000.sat, bobFundingAmount = 10_000.sat, alicePushAmount = 0.msat, bobPushAmount = 0.msat)
+        val (alice, bob) = reachNormal(aliceFundingAmount = 100_000.sat, bobFundingAmount = 10_000.sat, alicePushAmount = 0.msat, bobPushAmount = 0.msat)
         val leaseRate = LiquidityAds.LeaseRate(0, 0, 100 /* 5% */, 1.sat, 200, 100.msat)
         run {
             val liquidityRequest = LiquidityAds.RequestRemoteFunding(1_000_000.sat, bob.currentBlockHeight, leaseRate)
             assertEquals(10_001.sat, liquidityRequest.rate.fees(FeeratePerKw(1000.sat), liquidityRequest.fundingAmount, liquidityRequest.fundingAmount).total)
             val cmd = ChannelCommand.Commitment.Splice.Request(CompletableDeferred(), null, null, liquidityRequest, FeeratePerKw(1000.sat))
-            val (_, actions1) = bob.process(cmd)
-            assertTrue(actions1.isEmpty())
+            val (bob1, actionsBob1) = bob.process(cmd)
```

Comment:
nit: we can use `reachQuiescent`

Reply:
`reachQuiescent` expects and returns a `SpliceInit` message, so it can't be used for splice failures.